lms/djangoapps/learner_recommendations/utils.py

```diff
@@ -55,6 +55,67 @@ def _remove_user_enrolled_course_keys(user, course_keys):
     return enrollable_course_keys
 
 
+def _has_country_restrictions(product, user_country):
+    """
+    Helper method that tell whether courses or programs have any country restrictions.
+    Args:
+      product: course/program
+      user_country (string): country the user is logged in from
+
+    Returns:
+        True if the product is restricted in the country and False otherwise
+    """
+    if not user_country:
+        return False
+
+    allow_list, block_list = [], []
+    location_restriction = product.get("location_restriction", None)
+    if location_restriction:
+        restriction_type = location_restriction.get("restriction_type")
+        countries = location_restriction.get("countries")
+        if restriction_type == "allowlist":
+            allow_list = countries
+        if restriction_type == "blocklist":
+            block_list = countries
+
+    return user_country in block_list or (allow_list and user_country not in allow_list)
+
+
+def _parse_course_owner_data(owner):
+    """
+    Helper to parse course owner data.
+    """
+    return {
+        "key": owner.get("key"),
+        "name": owner.get("name"),
+        "logo_image_url": owner.get("logo_image_url")
+    }
+
+
+def course_data_for_course_about_page_recommendations(course_data):
+    """Helper to parse course data for course about page recommendations"""
+    course_runs = course_data.get("course_runs")
+    recommended_course_data = {}
+    if course_runs:
+        active_course_run = [course_run for course_run in course_runs
+                             if course_run.get("availability") == "Current"][0]
```

Comment:
Can we reduce the "if" conditions? 
```suggestion
    recommended_course_data = {}
    if course_runs:
        active_course_run = [course_run for course_run in  course_data.get("course_runs", [])
                             if course_run.get("availability") == "Current"][0]
```

Reply:
If any course has no course runs, the code can be broken in this case that's why added one extra if condition to handle this scenario.

Reply:
You can add a default value like I added `course_data.get("course_runs", [])`